### PR TITLE
BookB-Pset_GeoScienceFeatureTypeFold

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_GeoScienceFeatureTypeFold

## Description: 

Properties for folds.

| Property | Datatype | Description |
|----------|----------|-------------|
|ProfileType|IfcIdentifier|Contains a term from a controlled vocabulary specifying the concave/convex geometry of fold relative to earth surface, and relationship to younging direction in folded strata if known. (e.g., antiform, synform, neutral, anticline, syncline, monocline, ptygmatic).|
|Strike|IfcPlaneAngleMeasure|The geographic direction of a line created by the intersection of a plane and the horizontal.|
|Quadrant|IfcPlaneAngleMeasure|The dip orientation. |
|DipDirection|IfcPlaneAngleMeasure|The geographic  direction of a line created by the intersection of a plane and the horizontal plane. If non specific convention is used, this angular value is in the range 0 to 18 degrees. In this case, there is an ambiguity on the dip orientation. For example, a plane with an orientation of 90 degrees from the nord could either have a dip direction to the North or to the South. Thus, to avoid this ambiguity, a strike value is generally completed with an indication of the dip orientation (Quadrant).|

### CODE REVIEW

File: IFC4x3/Properties/q/Quadrant_2KShyZ_Xf6twsGaPpAX7HT/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Quadrant_2KShyZ_Xf6twsGaPpAX7HT" Name="Quadrant" UniqueId="9472bf23-fa1a-46df-ad90-919cca84745d" PrimaryDataType="IfcPlaneAngleMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/q/Quadrant_2KShyZ_Xf6twsGaPpAX7HT/DocProperty.xml]: This property name and description do not seem to match.

File: IFC4x3/Properties/d/DipDirection_17WEfvK3L9Ehv8Rdn5AK_c/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DipDirection_17WEfvK3L9Ehv8Rdn5AK_c" Name="DipDirection" UniqueId="4780ea79-5035-493a-be48-6e7c45294fa6" PrimaryDataType="IfcPlaneAngleMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/d/DipDirection_17WEfvK3L9Ehv8Rdn5AK_c/DocProperty.xml]: Re-using, right?

File: IFC4x3/Properties/s/Strike_2QDp8uDwH5yOXoaLbKFWum/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The geographic direction of a line created by the intersection of a plane and the horizontal.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/Strike_2QDp8uDwH5yOXoaLbKFWum/Documentation.md:1]: The angle is between true north and the created line? Measured CCW or CW?

**Review by SergejMuhic — COMMENTED** [review]
With types, now it would probably make sense to use IfcTypeObject.

File: IFC4x3/Properties/q/Quadrant_2KShyZ_Xf6twsGaPpAX7HT/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/q/Quadrant_2KShyZ_Xf6twsGaPpAX7HT/DocProperty.xml]: > This property name and description do not seem to match.

Agreed! @j-weil

File: IFC4x3/Properties/p/ProfileType_1AEKQiP7PFbuAGZ6obzmNO/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Contains a term from a controlled vocabulary specifying the concave/convex geometry of fold relative to earth surface, and relationship to younging direction in folded strata if known. (e.g., antiform, synform, neutral, anticline, syncline, monocline, ptygmatic).
```
Comment (pjanck) [inline IFC4x3/Properties/p/ProfileType_1AEKQiP7PFbuAGZ6obzmNO/Documentation.md]:
```suggestion
Contains a term from a controlled vocabulary specifying the concave/convex geometry of fold relative to earth surface, and relationship to younging direction in folded strata if known (e.g., antiform, synform, neutral, anticline, syncline, monocline, ptygmatic).
```

File: IFC4x3/Properties/s/Strike_2QDp8uDwH5yOXoaLbKFWum/Documentation.md (hunk 3 above)
Comment (pjanck) [reply IFC4x3/Properties/s/Strike_2QDp8uDwH5yOXoaLbKFWum/Documentation.md:1]: Geographic directions are (usually?) measured CW from North.

**Review by pjanck — APPROVED** [review]
Reviewed verbally with @larswik.

Note for the future: a unification of properties like Quadrant would be beneficial :)

File: IFC4x3/Properties/q/Quadrant_2KShyZ_Xf6twsGaPpAX7HT/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/q/Quadrant_2KShyZ_Xf6twsGaPpAX7HT/DocProperty.xml]: Removed. Didn't see the purpose with both DipDirection and StrikeDirection.

**Pull request merged**